src/battle-tooltips.ts

```diff
@@ -296,8 +298,10 @@ class BattleTooltips {
 						index++;
 					}
 				}
+				tooltipType = 'pokemon';
 			} else {
 				buf = this.showPokemonTooltip(pokemon);
+				tooltipType = 'pokemon';
 			}
```

Comment:
```suggestion
			} else {
				buf = this.showPokemonTooltip(pokemon);
			}
			tooltipType = 'pokemon';
```

These can be combined.